Client/Frontend/Settings/CustomSearchViewController.swift

```diff
@@ -50,9 +49,11 @@ class CustomSearchViewController: SettingsTableViewController {
         super.viewDidLoad()
         title = .SettingsAddCustomEngineTitle
         view.addSubview(spinnerView)
-        spinnerView.snp.makeConstraints { make in
-            make.center.equalTo(self.view.snp.center)
-        }
+        spinnerView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            spinnerView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            spinnerView.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        ])
```

Comment:
This is great. However, we should add these in a separate function called `setupConstraints` and call that from `viewDidLoad()` as this is the pattern we are generally switching to throughout the app. :)

Reply:
For sure. Please see [26549b3](https://github.com/mozilla-mobile/firefox-ios/pull/17089/commits/26549b325f8e4f0b28457c4027dff517fae17c22)